conf/cassandra.yaml

```diff
@@ -1752,6 +1752,17 @@ drop_compact_storage_enabled: false
 # write_consistency_levels_warned: []
 # write_consistency_levels_disallowed: []
 #
+# Guardrail to warn or fail when writing column values larger than threshold.
+# This guardrail is only applied to the values of regular columns because both the serialized partitions keys and the
+# values of the components of the clustering key already have a fixed, relatively small size limit of 65535 bytes, which
```

Comment:
That method was moved from `FBUtilities` to `ByteBufferUtil` by CASSANDRA-1714, in times of 0.7.0-rc4. I have updated the comment to point to the right methods.

As for the whole explanation about where that limitation in size comes from, I don't really now the full history but it seems it was introduced by CASSANDRA-767 for 0.7.0-beta1. That was 12 years ago, while adopting byte arrays for representing keys instead of Java strings.